csharp-basics/exercises/Polymorphism/Hierarchy/Food.cs

```diff
@@ -0,0 +1,17 @@
+namespace Hierarchy
+{
+    public abstract class Food
+    {
+        public int Quantity { get; set; }
```

Comment:
Kāpēc šis ir public?

Reply:
![image](https://user-images.githubusercontent.com/106081841/184018761-ba8f91f3-962a-45ea-a786-5aed70fedd67.png)